lib/health.js

```diff
@@ -55,13 +55,16 @@ module.exports.exposeHealthEndpoint = (server, endpoint = '/health', framework =
       break;
 
     case 'express':
-      server.get(endpoint, (req, res, next) => {
+      server.get(endpoint, (req, res) => {
         healthHandler(req.query.filter)
           .then(({ statusCode, status }) => {
-            res.status(statusCode).send(status);
-            next();
+            res.status(statusCode).json(status);
           })
-          .catch(err => next(err));
+          .catch(err => {
+            const errorCode = err.name.replace('Error', '');
+
+            res.status(err.statusCode).json({ code: errorCode, message: err.message });
```

Comment:
Hmm I think you can pass the error directly, BaseError (https://github.com/AugustHome/server-health/blob/master/lib/errors/base_error.js#L38) override the toJSON method so I guess when express call `JSON.stringify` we will return the same object? Isn't express using https://github.com/pillarjs/finalhandler#finalhandlerreq-res-options? Because calling `next(err)` should work?

Reply:
true, that works.